cats-effect/src/main/scala/github4s/cats/effect/Implicits.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2016-2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package github4s.cats.effect
+
+import cats.effect.IO
+import github4s.HttpRequestBuilderExtensionJVM
+import github4s.free.interpreters.{Capture, Interpreters}
+import scalaj.http.HttpResponse
+
+object implicits extends HttpRequestBuilderExtensionJVM {
```

Comment:
By now, I think that you should. Nevertheless, I guess we want to get rid of this kind of implicit import.

https://github.com/47deg/github4s/issues/45

One option would use a fully compatible http client for both jvm and js: https://github.com/pepegar/hammock . 

What do you think?